README.md

```
@@ -1,97 +1,89 @@
-You've successfully run and modified your React Native App. :partying_face:
-
-### Now what?
+<br/>
+<br/>
 
-- If you want to add this new React Native code to an existing application, check out the [Integration guide](https://reactnative.dev/docs/integration-with-existing-apps).
-- If you're curious to learn more about React Native, check out the [docs](https://reactnative.dev/docs/getting-started).
+# 협업 규칙
+## 브랜치 전략 (Branch Strategy)
+- **GitHub 브랜치 전략**:
+    - Git Flow (main, develop, feature, release, hotfix)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)